var/spack/repos/builtin/packages/neurodamus-core/package.py

```diff
@@ -12,7 +12,8 @@ class NeurodamusCore(Package):
     git      = "ssh://bbpcode.epfl.ch/sim/neurodamus-core"
 
     version('develop', git=git, branch='master')
-    version('2.2.1', git=git, tag='2.2.1', preferred=True)
+    version('2.3.0', git=git, tag='2.3.0', preferred=True)
```

Comment:
what is changed in the code? just curious..

Reply:
It's the version with the different handling of ModOverride